src/main/java/bio/terra/service/resourcemanagement/google/GoogleResourceService.java

```diff
@@ -572,6 +586,55 @@ public void enableIamPermissions(Map<String, List<String>> userPermissions, Stri
         throw new EnablePermissionsFailedException("Cannot enable iam permissions", lastException);
     }
 
+    public void removeIamPermissions(Map<String, List<String>> userPermissions, String projectId)
+        throws InterruptedException {
+
+        GetIamPolicyRequest getIamPolicyRequest = new GetIamPolicyRequest();
+
+        Exception lastException = null;
+        int retryWait = INITIAL_WAIT_SECONDS;
+        for (int i = 0; i < RETRIES; i++) {
+            try {
+                CloudResourceManager resourceManager = cloudResourceManager();
+                Policy policy = resourceManager.projects()
+                    .getIamPolicy(projectId, getIamPolicyRequest).execute();
+                final List<Binding> bindingsList = policy.getBindings();
+
+                // Remove members from the current policies
+                for (Map.Entry<String, List<String>> entry : userPermissions.entrySet()) {
+                    CollectionUtils.transform(bindingsList, b -> {
```

Comment:
Maybe I don't understand what this is doing, but it doesn't look like this `transform` changes a binding, other than calling setters on it. If so, using `foreach` would be better IMO.

Reply:
Transform actually mutates the list that you pass in.  It's a strange interaction but that's the best I could find with the commons lib.

Reply:
Yes, but in this case there is no mutation, only side effects, so transform is not needed.

Reply:
Ah yeah, I see your point.  Fair enough